### ISSUE TITLE
Restrict extra indent in brackets

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#354]
# Rule request

## Thesis

We need to be strict about the maximum indent in all brackets. Here's an example of code that currently does not raise any violations, but should:

```python
some(
                                    1,
                                    2,
                                    3,
)
```

Note how many spaces are there.

## Reasoning

Extra indentation is not required and is bad for readability.
